src/itest/resources/features/company-links-company-profile-errors.feature

```diff
@@ -23,4 +23,23 @@ Feature: Process company profile links for error scenarios
     And "persons-with-significant-control" exist for company "00006400"
     When A valid "changed" message is consumed from the "company-profile" stream
     Then The message is placed on the "retry" topic
-    And The message is placed on the "error" topic
\ No newline at end of file
+    And The message is placed on the "error" topic
+
+  Scenario: Process message when the api returns 503
+    Given Company links consumer is available
+    And The company profile api is unavailable
+    And "persons-with-significant-control" exist for company "00006400"
+    When A valid "changed" message is consumed from the "company-profile" stream
+    Then the message should retry 3 times on the company-profile topic and then error
+
+  Scenario: Process message when the API returns 409
```

Comment:
Test lacks a 'then' step, what are we actually testing here?

Reply:
I would have thought we would want to confirm that it would not retry and would go the the invalid queue as it should throw a non retryable